achievements/models.py

```diff
@@ -2,5 +2,97 @@
 from __future__ import unicode_literals
 
 from django.db import models
-
+from registration.models import UserInfo
 # Create your models here.
+
+ACHIEVEMENT_CHOICE = (('article','Article'),
+                      ('contribution','Contribution'),
+                      ('gsoc','GSoC'),
+                      ('intern','Internship'),
+                      ('speaker','Speaker'),
+                      ('contest','contest'),
+                      ('scholarship', 'Scholarship')
+                      ('other','Other'))
+INTERN_CHOICE = (('internship','Internship'),
+                 ('masters','Masters'),
+                 ('exchange student','Exchange programme'))
+SPEAKER_CHOICE =(('talk',' Talk'),
+                 ('demo','Demo'),
+                 ('workshop','Workshop'),
+                 ('paper','Paper Presentation'),
+                 ('other','Other'))
+class Achievement(models.Model):
+    achievement_id = models.BigIntegerField( primary_key=True, blank=False, unique=True)
+    achieve_type = models.CharField(max_length=15, choices=ACHIEVEMENT_CHOICE)
+    username = models.ForeignKey(UserInfo, blank=False, null=False)
+
+
+class Article(models.Model):
+    achievement_id = models.ForeignKey(Achievement, blank=False, null=False)
+    username = models.ForeignKey(UserInfo, blank=False, null=False)
+    magazine_name = models.CharField(max_length=50, blank=False, null=False)
+    publication_date = models.DateField(blank=False)
+
+
+class Contribution(models.Model):
+    achievement_id = models.ForeignKey(Achievement, blank=False, null=False)
+    bug_id = models.BigIntegerField( blank=False)
```

Comment:
Increase max_length to at least 200 for future uses. remove blank=Flase and null=False from every Field. No need to so that as by default its  blank=Flase and null=False